spharpy/samplings/samplings.py

```diff
@@ -352,6 +352,68 @@ def eigenmike_em32():
     return SamplingSphere.from_spherical(rad, theta, phi)
 
 
+def eigenmike_em64():
+    """Microphone positions of the Eigenmike em64 by mhacoustics.
+
+    according to
+    the Eigenmuke user manual on the homepage [#]_.
```

Comment:
Maybe link directly - or make the reference a link. And theres a typo in Eigenmike
```suggestion
    the `Eigenmike user manual <link URL>`_.
```